fix(DebtsManager): correct debt summary calculations in DetailSharedD…

MarvinFos opened the pull request:
…ebts

Fix incorrect calculations for pending and arrears counts in debt summary cards. Update the logic to properly calculate pending amounts based on total, collected and arrears values.

MarvinFos committed:
fix(DebtsManager): correct debt summary calculations in DetailSharedDebts

Fix incorrect calculations for pending and arrears counts in debt summary cards. Update the logic to properly calculate pending amounts based on total, collected and arrears values.

elvisfos reviewed (dismissed):
## CODE REVIEW REPORT - Github Projects
**PR:** # [PR](https://github.com/mariogfos/condaty2025-admin/pull/267) - [Github Dashboard issue](https://github.com/Condaty/admin/issues/21)
**App:** Admin  
**Autor:** @MarvinFos 
**Revisor:** @elvisfos 
**Fecha:** 05/12/25  
**Estado:**  **APROBADO**

---

## PROBLEMAS CRÍTICOS (BLOCKERS)
[x] **Seguridad:** Todo correcto
[x] **Funcionalidad rota:** No se detectan problemas
[x] **Performance crítica:** Sin incidencias

---

## ISSUES A CORREGIR
### **Alta Prioridad** (Merge bloqueado)
[x] Ningún issue detectado

### **Media Prioridad** (Corregir antes de deploy)
[x] Ningún issue detectado

### **Baja Prioridad** (Mejora futura)
[x] Ninguna sugerencia pendiente

---

## CHECKLIST RÁPIDO (Solo marcar problemas)
**Calidad de código:**
[x] Nombres claros
[x] Sin código duplicado
[x] Lógica sencilla

**Front específico:**
[x] Sin props drilling excesivo
[x] Sin re-renderizados innecesarios
[x] Accesibilidad correcta

**Seguridad:**
[x] Inputs validados
[x] Sin tokens/keys expuestos
[x] Permisos correctos

**Tests:**
[x] Tests completos para nueva lógica
[x] Coverage estable

---

## DECISIÓN FINAL
**Veredicto:** APROBADO  
**Acciones inmediatas:**  
[x] No se requieren correcciones  
[x] Listo para merge
# .github/workflows/review-metrics.yml
name: Code Review Metrics
on:
  pull_request_review:
    types: [submitted]

jobs:
  collect-metrics:
    runs-on: ubuntu-latest
    steps:
      - name: Extract Review Data
        id: review-data
        run: |
          # Script simple para métricas básicas
          echo "REVIEWER=${{ github.event.review.user.login }}" >> $GITHUB_ENV
          echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
      
      - name: Post Summary Comment
        uses: actions/github-script@v6
        with:
          script: |
            // Resumen automático en comentario
            github.rest.issues.createComment({
              issue_number: context.issue.number,
              owner: context.repo.owner,
              repo: context.repo.repo,
              body: 📊 **Métricas automáticas de review**\n\n✅ Issues reportados: ${{ github.event.review.state }}\n👤 Revisor: @${{ github.event.review.user.login }}\n📅 Fecha: ${new Date().toLocaleDateString()}
            })

elvisfos commented on src/modulos/DebtsManager/TabComponents/SharedDebts/DetalleDeudaCompartida/DetailSharedDebts.tsx:
@MarvinFos
Quitar y/o Remover los console.log()
@marcelofos 
Sus revisiones no debe pasar los console.log()

MarvinFos committed:
refactor(DebtsManager): remove debug console logs from components

Clean up code by removing unnecessary console.log statements used for debugging in Forgiveness, RenderForm and DetailSharedDebts components

MarvinFos commented:
corregido

elvisfos reviewed (dismissed):
src/modulos/DebtsManager/TabComponents/SharedDebts/DetalleDeudaCompartida/DetailSharedDebts.tsx

@MarvinFos @marcelofos 
este archivo sigue imprimiendo los console.lgo

MarvinFos committed:
refactor(DebtsManager): clean up commented code and improve form handling

remove unused commented code in Forgiveness component
clean up console logs and comments in DetailSharedDebts
improve form state management in RenderForm

MarvinFos committed:
Merge branch 'dev' of github.com:mariogfos/condaty2025-admin into marvin-fix/amount-debts

EstebanFOS commented on example.env.development:
NO SE DEBE DE ELIMINAR ESTE ARCHIVO

elvisfos committed:
file restaured
